code/index.html

```diff
@@ -3,28 +3,143 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>dogGO</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://use.typekit.net/epf8jwq.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <style></style>
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+    <!--HEADER SECTION-->
+    
+    <div class="navbar-container">
+      <header class="navbar">
+        <div class="hamburger-wrapper">
+          <div class="hamburger">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+        </div>
+        <h2 class="logo">     
+          <img src="./images/logo.png" alt="DogGO logo image">
+        </h2>
+        <div class="menu-container">
+          <nav>
+            <ul class=nav-menu>
+              <li><a href="#">Dog-stinations</a></li>
+              <li><a href="#">Off-Leash Experiences</a></li>
+              <li><a href="#">Booking</a></li>
+              <li><a href="#">A-BARK Us</a></li>
+              <li><a href="#">SIGN IN</a></li>
+            </ul>
+          </nav> 
```

Comment:
I think you can omit the `ul` and `li` elements, but if you want to keep them - remember to give nested elements new lines and indent

Reply:
It looks better with just <a> I think. :)